charts/ceph-csi-rbd/values.yaml

```diff
@@ -268,6 +273,11 @@ provisioner:
     ## https://github.com/kubernetes-csi/external-attacher#command-line-options
     extraArgs: []
 
+    args:
```

Comment:
please update helm documentation for new args

Reply:
Done